### ISSUE TITLE
MIME type / extension issues with `.rng` files

### ISSUE THREAD

**Issue by csae8092** [acdh-oeaw/repo-file-checker#20]
running filechecker on https://github.com/KfNGOe/ferdinand-I-data/blob/main/src/tei2dtabf/schema/basisformat_all_ferdinand.rng throws following errors:

```
"/data";"basisformat_all_ferdinand.rng";"ERROR";"File content doesn't match extension";"Extension: rng, MIME type: application/xml, MIME types allowed for this extension: "
"/data";"basisformat_all_ferdinand.rng";"ERROR";"File format not accepted";"MIME application/xml: preferred, extension rng: not accepted"
```

**Comment by zozlak** [comment]
We have two issues here:

1. General - the RNG format is not marked as ARCHE-compliant in https://github.com/acdh-oeaw/arche-assets/blob/master/AcdhArcheAssets/formats.json (in fact is not even mentioned there). This is to be discussed and fixed by Martina (or whoever takes over the maintenance of this list when Martina is on the leave).
2. RNG format recognition. Recognizing text formats is hard as virtually all of them are built on (a stack of) other formats and as a such are also valid documents in all those formats. E.g. here the RNG document is also a valid application/xml document. Checking the whole stack of formats is hard and resource-consuming so broad-application format-recognition libraries typically only recognize the "top-level" text format from the stack (as we see here `application/xml`) and if we want to dig deeper, this has to be implemented separately (either using libraries dedicated for a given format or by hand). For this reason I only make effort of implementing that for formats which are officially supported by ARCHE (see point one).

Summing up:

* As for now it's not a bug, it's a feature
* You can solve it either by renaming the file to `.xml` or by contacting Martina to register the RNG format in the list of ARCHE accepted ones and then get back here to request its correct recognition.
